spec/ddtrace/workers/loop_spec.rb

```diff
@@ -201,12 +204,12 @@
           expect { loop_back_off! }
             .to change { worker.loop_wait_time }
             .from(described_class::BASE_INTERVAL)
-            .to(described_class::BACK_OFF_RATIO)
+            .to(described_class::BASE_INTERVAL * described_class::BACK_OFF_RATIO)
```

Comment:
These tests were incorrect, but they just so happen to work because `BASE_INTERVAL` was 1, thus the calculation of backoff was trivially `1 * BACK_OFF_RATIO`.